tests/collection/collection-search.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { test } from '../fixtures';
+import { SearchOption } from '../models';
+
+test('Collection search metadata', async ({ collectionPage }) => {
+  await test.step(`Select "Search metadata"`, async () => {
+    await collectionPage.searchPage.clickSearchInputOption(
+      SearchOption.METADATA,
+    );
+  });
+
+  await test.step(`Query for "radio"`, async () => {
+    await collectionPage.searchPage.queryFor('radio');
+  });
+
+  await test.step(`Total result count is displayed`, async () => {
+    await collectionPage.collectionFacets.checkResultCount();
+  });
+
+  await test.step(`Results are displayed in display area - checking first item displayed`, async () => {
+    await collectionPage.infiniteScroller.checkFirstItemResult();
+  });
```

Comment:
same question I have for this test block - can we roll up these 2 tests into 1? it makes much more sense when they are grouped together as opposed to each piece of the flow as a function. let's talk about this organizational step, is that technically feasible?

Reply:
yes it's feasible that we can have multiple functions in 1 test step.
i structured it this way so it will be easy to check/detect in reporting logs at which step caused the test to fail.